pkg/dssproto/dss.proto

```diff
@@ -211,13 +211,18 @@ message LatLngPoint {
 
 // Parameters of a message informing of new full information for an Identification Service Area.  Pushed (by a client, not the DSS) directly to clients with subscriptions when another client makes a change to airspace within a cell with a subscription.
 message PutIdentificationServiceAreaNotificationParameters {
+  // The new or updated extents of the Identification Service Area.
+  // 
+  // Omitted if Identification Service Area was deleted.
+  Volume4D extents = 1;
+
   // Identification Service Area that the notifying client changed or created.
   // 
   // If this field is populated, the Identification Service Area was created or updated.  If this field is not populated, the Identification Service Area was deleted.
-  IdentificationServiceArea service_area = 1;
+  IdentificationServiceArea service_area = 2;
```

Comment:
Not a problem for this PR, but we'll have to note that gRPC message indices aren't guaranteed to stay constant -- that's atypical for a gRPC approach but I think is acceptable here because gRPC is just behind the HTTP bridge.

Reply:
This actually is a breaking change. So previous versions will not be compatible with this version. It's early days, so we could agree to take some down time, but I think we should probably just keep the indices constant.

Reply:
Note that this file is auto-generated. I *think* we might have to adjust the upstream API to alter the member ordering. Will investigate how the indices are determined.

Reply:
At some point we also might want to consider abandoning auto generation for reasons like this

Reply:
A one-time breaking change should be ok for this PR because there won't be any impact.  We can discuss further following this PR.